src/SDKs/SqlManagement/Management.Sql/Generated/Models/SecurityAlertPolicyUseServerDefault.cs

```diff
@@ -27,4 +27,32 @@ public enum SecurityAlertPolicyUseServerDefault
         [EnumMember(Value = "Disabled")]
         Disabled
     }
+    internal static class SecurityAlertPolicyUseServerDefaultEnumExtension
+    {
+        internal static string ToSerializedValue(this SecurityAlertPolicyUseServerDefault? value )  =>
+            value == null ? null : (( SecurityAlertPolicyUseServerDefault )value).ToSerializedValue();
+
+        internal static string ToSerializedValue(this SecurityAlertPolicyUseServerDefault value )
+        {
+            switch( value )
+            {
+                case SecurityAlertPolicyUseServerDefault.Enabled:
+                    return "Enabled";
+                case SecurityAlertPolicyUseServerDefault.Disabled:
+                    return "Disabled";
+            }
+            return null;
+        }
+
+        internal static SecurityAlertPolicyUseServerDefault? ParseSecurityAlertPolicyUseServerDefault( this string value )
+        {
+            switch( value )
+            {
+                case "Enabled":
+                    return SecurityAlertPolicyUseServerDefault.Enabled;
+                case "Disabled":
+                    return SecurityAlertPolicyUseServerDefault.Disabled;            }
```

Comment:
Filed https://github.com/Azure/autorest/issues/2386 against autorest code generator to fix this